drift/views/v1.py

```diff
@@ -46,6 +46,16 @@ def comparison_report():
         raise HTTPError(HTTPStatus.BAD_REQUEST, message=error.message)
 
 
+def _is_mgmt_url(path):
+    """
+    small helper to test if URL is for management API.
+    """
+    if path.startswith('/mgmt/'):
```

Comment:
`startswith` returns a boolean so you can simplify this to `return path.startswith('/mgmt')` I think.